src/main/java/uk/gov/hmcts/divorce/divorcecase/util/AddressUtil.java

```diff
@@ -15,6 +20,11 @@ public final class AddressUtil {
 
     private static final String COMMA_SEPARATOR = ",";
     private static final int ADDRESS_LINE_MAX_CHARS = 25;
+    private static final List<String> UK_TERMS = Arrays.asList("unitedkingdom", "uk", "england", "wales");
```

Comment:
should "united kingdom" not be included?

Reply:
if they set "United Kingdom" as country, and then use an English / Welsh postcode, isEnglandOrWales will return true. This should be allowed cos postcode lookup sets country as United Kingdom not England / Wales